### ISSUE TITLE
Unable to add bluetooth to the config

### ISSUE THREAD

**Issue by boblablaw** [pgaskin/NickelMenu#163]
Whenever I try to add bluetooth to the config I get a config error.  Other options seem to work fine.

Here is my config:
menu_item :main			:Sketch Pad			:nickel_extras      :sketch_pad
menu_item :main			:Enable Bluetooth		:nickel_bluetooth   :enable
menu_item :main			:Bluetooth Patch (toggle)	:cmd_output         :500:quiet :rmmod -w /mnt/onboard/.adds/kobo-libra2-uhid-module/uhid.ko
  chain_success			:skip:3
  chain_failure			:cmd_spawn          		:quiet		     :exec /mnt/onboard/.adds/kobo-libra2-uhid-module/run.sh
  chain_success			:dbg_toast          		:insmod uhid.ko successfully
  chain_always			:skip				:-1
  chain_success			:dbg_toast          		:rmmod -w uhid.ko successfully

**Comment by inklesspen** [comment]
The bluetooth enable feature was added in December 2023, but the most recent NickelMenu release is from January 2022.

**Comment by pgaskin** [comment]
Yeah, I'll get to it soon, but I'm currently very busy, and will also be working on kobopatch first. I also don't have any Kobo device with bluetooth, which makes it harder for me to test.

**Comment by inklesspen** [comment]
fwiw if me and my clara 2e (and clara color) can be helpful here, let me know.

**Comment by ycx0303** [comment]
> Yeah, I'll get to it soon, but I'm currently very busy, and will also be working on kobopatch first. I also don't have any Kobo device with bluetooth, which makes it harder for me to test.

Please add bluetooth control feature, that would be very useful for me! Thanks!

**Comment by jadehawk** [comment]
So safe to assume the "nickel_bluetooth" in the DOC on the [website](https://github.com/pgaskin/NickelMenu/blob/v0.5.4/res/doc) is a future "feature". I notice that setting is not in the actual DOC in the release we can download.

Thank you for this tool!!

**Comment by pgaskin** [comment]
It's not released yet. You can download a build from GitHub Actions.
